src/data/locations.json

```diff
@@ -0,0 +1,14 @@
+[
+    {
+      "storeType": "Food Pantry 1",
```

Comment:
This are good for starting off, eventually we will have an API endpoint with the database to pull this dataset from.